### DISCUSSION TITLE
globalThis is not defined when adding angularfire modules to Angular Universal app

### DISCUSSION THREAD

**Discussion by chrisoppedal** [angular/angularfire#2716] [Q&A]
Angular version : 11.0.4

Firebase:  Latest
AngularFire: version 6.1.4

Node version: 12.18.3

I started a new project with these commands:
ng new testProject
ng add @nguniversal/express-engine
ng add @angular/fire
ng deploy
- The project works with ng serve and ng deploy
- add an AngularFire module (AngularFireAuthModule) to app.module.ts
ng deploy
- The app runs locally with ng serve but ng deploy fails to publish the 'ssr' function because of 'globalThis is not defined' error in firebase function logs.

More debugging info: https://stackoverflow.com/questions/65243054/cant-update-firebase-function-to-node-12-in-angular-universal-app-with-angularf

**Comment by Sangarllo** [comment]
This problem (globalThis is not defined...) is solved if you use Node V14 LTS version. I have the same problem when I use Node 10 or Node 12. Good luck!
